Sources/Core/API/Label.swift

```diff
@@ -157,6 +160,15 @@ public final class Label: SceneObject, InstanceHashable, ActionPerformer, Plugga
         autoResize()
     }
 
+    private func scaleDidChange(from oldValue: Metric) {
+        guard oldValue != scale else {
+            return
+        }
+
+        layer.scale = scale
+        autoResize()
```

Comment:
You shouldn't need to auto resize anymore since we are now using the layer's scale.

Reply:
Right. The size doesn't really change. Fixed.